graph-proxy/src/graphql/workflows.rs

```diff
@@ -318,6 +362,22 @@ impl Task {
     async fn dependencies(&self) -> Vec<String> {
         self.node_status.children.clone()
     }
+
+    /// Artifacts produced by a task
+    async fn artifacts(&self) -> Vec<Artifact> {
+        match self.node_status.outputs.as_ref() {
+            None => vec![],
```

Comment:
This it a nit-pick, but we usually use `Vec::new` for empty Vecs

Reply:
No worries, I wasn't aware of the convention, that's useful to know